README.md

```diff
@@ -1,25 +1,27 @@
-<img src="imgs/logo.png" width="450">
+<img src="https://raw.githubusercontent.com/goodwithtech/dockle/master/imgs/logo.png" width="450">
```

Comment:
I think better to use a relative path.

Reply:
> I think better to use a relative path.

I should've mentioned about the real path, sorry. I changed to the real path so the README on your Docker Hub's repo displays the images.
https://hub.docker.com/r/goodwithtech/dockle